include/swift/AST/DiagnosticsSema.def

```diff
@@ -2446,6 +2446,10 @@ ERROR(no_MaxBuiltinIntegerType_found,none,
    "standard library error: _MaxBuiltinIntegerType is not properly defined", ())
 ERROR(no_MaxBuiltinFloatType_found,none,
    "standard library error: _MaxBuiltinFloatType is not properly defined", ())
+ERROR(integer_literal_overflows_maxwidth, none,
+   "Integer literal size limit exceeded: '%2' needs %1 bits, "
```

Comment:
Minor thing: diagnostics usually don't start with upper-case, the compiler itself is all lower-case and then Xcode will do the upper casing if necessary.